### PR TITLE
[FIX] crm: fix undesired meetings via partner's smart button

### PR DESCRIPTION
**Pull request by kitan191**
- Create a Meeting for Contact X
- Go to Contacts and open Contact X
The Meetings smart button shows 1, but when clicking on it,
all user & partner meetings are displayed.

opw-2320299

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by kitan191** [comment]
ping @nim-odoo

File: addons/crm/models/res_partner.py [hunk 1]
```diff
@@ -50,12 +50,8 @@ def _compute_meeting_count(self):
             partner.meeting_count = len(partner.meeting_ids)
 
     def schedule_meeting(self):
-        partner_ids = self.ids
-        partner_ids.append(self.env.user.partner_id.id)
         action = self.env.ref('calendar.action_calendar_event').read()[0]
-        action['context'] = {
```
Comment (nim-odoo) [inline addons/crm/models/res_partner.py:56]: I'm not sure how this relates. `default_partner_ids` gives the default values when creating a new meeting, it doesn't impact the list of meetings displayed when clicking on the stat button.

**Review by nim-odoo — COMMENTED** [review]
I'd think `action['domain'] = [('id', 'in', self.meeting_ids.ids)]` is the only line necessary.

**Review by nim-odoo — APPROVED** [review]
robodoo delegate+

**Comment by kitan191** [comment]
robodoo r+

**Pull request closed without merging**